### PR TITLE
Update TokenManager.sol

### PR DESCRIPTION
**Pull request by cstrangedk**

### CODE REVIEW

File: proxy/contracts/MessageProxyForMainnet.sol [hunk 1]
```diff
@@ -214,6 +260,16 @@ contract MessageProxyForMainnet is Initializable {
         );
     }
 
+    /**
+     * @dev Posts incoming message to `srcChainID`.
```
Comment (payvint) [inline proxy/contracts/MessageProxyForMainnet.sol]: from srcChainID

File: proxy/contracts/predeployed/TokenManager.sol [hunk 2]
```diff
@@ -44,7 +44,14 @@ interface ILockAndDataTM {
 // This contract runs on schains and accepts messages from main net creates ETH clones.
 // When the user exits, it burns them
 
-
+/**
+ * @title Token Manager
+ * @dev Runs on SKALE Chains, accepts messages from mainnet, and instructs
+ * TokenFactory to create clones. TokenManager manages ETH, ERC20 and ERC721
```
Comment (payvint) [inline proxy/contracts/predeployed/TokenManager.sol]: I think is a duplication of the next sentense

File: proxy/contracts/predeployed/TokenManager.sol [hunk 3]
```diff
@@ -400,8 +413,18 @@ contract TokenManager is PermissionsForSchain {
         );
     }
 
-    // Receive money from main net and Schain
-
+    /**
+     * @dev Allows MessageProxy to post operational message to mainnet
```
Comment (payvint) [inline proxy/contracts/predeployed/TokenManager.sol]: from main net or SKALE chains

File: proxy/contracts/predeployed/TokenManager.sol [hunk 4]
```diff
@@ -400,8 +413,18 @@ contract TokenManager is PermissionsForSchain {
...
+     * or SKALE chains. Also used to refill mainnet SKALE chain agent to allow SKALE chain users
```
Comment (payvint) [inline proxy/contracts/predeployed/TokenManager.sol]: I think it is a copy of DepositBox contract. "postMessage" function could not refill main net SKALE chain agent

File: proxy/contracts/predeployed/TokenManager.sol [hunk 5]
```diff
@@ -443,13 +466,17 @@ contract TokenManager is PermissionsForSchain {
         }
     }
 
-    // This is called by schain owner.
-    // Exit to main net
+    /**
+     * @dev Performs an exit (post outgoing message) to Mainnet by SKALE chain owner.
```
Comment (payvint) [inline proxy/contracts/predeployed/TokenManager.sol]: by SKALE chain owner?

Comment (cstrangedk) [reply proxy/contracts/predeployed/TokenManager.sol]: Clarified.

File: proxy/contracts/predeployed/TokenManager.sol (hunk 4 above)
Comment (cstrangedk) [reply proxy/contracts/predeployed/TokenManager.sol]: Clarified.

File: proxy/contracts/predeployed/TokenManager.sol (hunk 3 above)
Comment (cstrangedk) [reply proxy/contracts/predeployed/TokenManager.sol]: Clarified.

File: proxy/contracts/predeployed/TokenManager.sol (hunk 2 above)
Comment (cstrangedk) [reply proxy/contracts/predeployed/TokenManager.sol]: Clarified.

File: proxy/contracts/MessageProxyForMainnet.sol (hunk 1 above)
Comment (cstrangedk) [reply proxy/contracts/MessageProxyForMainnet.sol]: Clarified.

**Review by payvint — APPROVED** [review]
Good

**Pull request merged**